dvc/objects/__init__.py

```diff
@@ -7,10 +8,22 @@
 logger = logging.getLogger(__name__)
 
 
-def save(odb, obj, **kwargs):
+def save(odb, obj, jobs=None, **kwargs):
```

Comment:
Other places in the codebase (e.g fetch) already pass `jobs=...` though `odb.add` doesn't use it (AFAIK). So we don't need to modify those